docker/ci/Dockerfile

```diff
@@ -8,17 +8,17 @@ RUN useradd -ms /bin/bash postgres
 USER postgres
 
 # install cargo pgx
-RUN cargo install cargo-pgx --version '^0.2'
+RUN cargo install cargo-pgx --version '^0.2' --root /pgx-0.2 \
+    && cargo install cargo-pgx --version '^0.4' --root /pgx-0.4
```

Comment:
note that [the CI scripts](https://github.com/timescale/timescaledb-toolkit/blob/8183d126e4898d64892ee358b3b48b8851f7abbf/.github/workflows/ci.yml) (well [here](https://github.com/timescale/timescaledb-toolkit/blob/main/tools/build)) will look for `cargo pgx` in the path, so we need to add at least one of these to the path to ensure they keep working. My reccomendation would be to add the `0.2` one then switch it in the same PR we update the extension to `0.4`

Reply:
(I don't love how many layers of indirection CI has, but I don't know of any good way to fix that 🤔)

Reply:
Good catch! Will update using your recommendation